diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigYamlTest.kt

```diff
@@ -36,6 +38,19 @@ inline class RulesConfigYamlTest(private val pathMap: Map<String, String> =
         checkComments("../diktat-analysis.yml")
     }
 
+    @Test
+    fun `check kotlin version`() {
```

Comment:
This test can be used to ensure that we have kotlin version set in all our configs; please add testing of version logic to `ConfigReaderTest` in `diktat-common`. You can simply add two cases: that retrieving version explicitly (configuration.get().kotlinVersion) works and that retrieving it from `CommonConfiguration` works too.